### PR TITLE
Setup React and Linters

### PR DESCRIPTION
**Pull request by initseis**
In this project I:

- Created a React project with the **create-react-app** command
- Setup Linters
- Configured gitignore file
- Tested the functionality
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  }
```
Comment (henatan99) [inline package.json:47]: - [ ] There should be `devDependencies` in your `package.json`, unless you made some mistake in your `react app` initialization. As a result, your app is not running correctly. Kindly fix this error and submit it for another review. :pray: 

image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @sevinchek , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (initseis) [reply package.json:47]: Hi Henok. I tested again by cloning the project in another folder and It's running ok.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

As explained by the CR, the npm packages were installed in your local machine, but since they weren't in the `package.json` files, anyone from a different machine wouldn't that doesn't have them would get these errors

I also highly recommend using the right [react linters](https://github.com/microverseinc/linters-config/tree/master/react-redux) we provide

Keep rocking 🚀!

**Pull request merged**